.buildkite/pipeline.yml

```diff
@@ -1,3 +1,57 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_MAGE_VERSION: '1.14.0'
+  SETUP_GOLANG_VERSION: '1.20.4'
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: ":golangci-lint: Checks formatting / linting"
+    command:
+      - ".buildkite/scripts/pre-install-command.sh 'mage -debug check'"
+    agents:
+      image: "golang:${SETUP_GOLANG_VERSION}"
```

Comment:
Oh! I didn't know that environment variables could be used to set the `image` key. Nice!